modules/core/04-channel/keeper/keeper.go

```diff
@@ -570,3 +570,14 @@ func (k Keeper) iterateHashes(ctx sdk.Context, iterator db.Iterator, cb func(por
 		}
 	}
 }
+
+// hasInflightPackets returns true if there are packet commitments stored and
+// false otherwise.
+//
+//lint:ignore U1000 Ignore unused function temporarily for debugging
+func (k Keeper) hasInflightPackets(ctx sdk.Context) bool {
+	iterator := sdk.KVStorePrefixIterator(ctx.KVStore(k.storeKey), []byte(host.KeyPacketCommitmentPrefix))
+	defer sdk.LogDeferred(ctx.Logger(), func() error { return iterator.Close() })
+
+	return iterator.Valid()
```

Comment:
Could re-use `iterateHashes` with an ugly closure but don't think its worth it. Additionally, logging the closing of the iterator as is done elsewhere.